library/rmc/gen_c_lib.c

```diff
@@ -0,0 +1,55 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+#include <assert.h>
+#include <limits.h>
+#include <math.h>
+#include <stdio.h>
+#include <stdbool.h>
+#include <string.h>
+
+// If a user assumes something, then it should hold
+void __CPROVER_assume(int condition) {
+    assert(condition);
+}
+
+// We can ignore atomics for simplicity
+void __CPROVER_atomic_begin(void) {
+}
+
+void __CPROVER_atomic_end(void) {
+}
+
+// These need to be manually defined for some reason
+double powi(double base, int expt) {
+    return pow(base, (double) expt);
+}
+
+float powif(float base, int expt) {
+    return (float) powi((double) base, expt);
+}
+
+// Used by cbmc invariants
+typedef bool __CPROVER_bool;
+
+#define OBJECT_SIZE(value) sizeof(*value)
+
+// A temporary definition to always cause checks to be true.
+#define POINTER_OBJECT(value) 0
+
+#define overflow(op, typ, var1, var2) \
+    (strcmp(op, "+") == 0) ? ( \
```

Comment:
instead of strcmp, you could `op[0] == '+'`

Reply:
Done.